.github/workflows/test.yml

```diff
@@ -6,8 +6,13 @@ jobs:
   test:
     runs-on: ubuntu-latest
 
+    strategy:
+      fail-fast: false
```

Comment:
Please explain the reason for this flag. What are the effects of this flag and why is this necessary.

Currently the `Check Stage Formatting` is exiting with code `2` but the stage itself shows as passing `✔`. Could it be caused by this?

Does setting `fail-fast: false` deceptively show all failing stages as passing?

Reply:
`fail-fast` description from [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast):
> When set to true, GitHub cancels all in-progress jobs if any matrix job fails. Default: true

Without it the `latest` job will be cancelled and not fully executed, because `dev` job failes
It can be seen in my fork [actions](https://github.com/Liklainy/dart/actions/runs/407909961) before I added `fail-fast: false`

Reply:
Part of `Check Stage Formatting` failed with code `2` in previous [actions](https://github.com/exercism/dart/runs/1498335845) too

Exit code of `fetchConfiglet` is swallowed [inside ](https://github.com/exercism/dart/blob/master/bin/check_formatting.dart#L8) `check_formatting` and job doesn't fail
The error in fetching script probably should be fixed in #275